docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -11,9 +11,38 @@ default:: `syn-ingress-nginx`
 The namespace in which to deploy this component.
 
 
-== Example
+== `release_name`
+
+[horizontal]
+type:: string
+default:: `ingress-nginx`
+
+The name ingress-nginx is deployed.
+Usually there is just one deployment and therefore no change is required.
+
+
+== `charts.ingress-nginx`
+
+[horizontal]
+type:: string
+default:: `3.21.0`
+
+
+== `helm_values`
+
+[horizontal]
+type:: yaml
```

Comment:
```suggestion
type:: dict
```

If I recall correctly, `dict` got used ad other places. Stating `yaml` would allow anything from string to array to dictionary.